index.bs

```diff
@@ -906,6 +930,10 @@ method is invoked, the user agent MUST:
 
 1. Let |clientDataHash| be the [=hash of the serialized client data=] represented by |clientDataJSON|.
 
+1. If the <code>|options|.{{CredentialRequestOptions/signal}}</code> is [=present=] and its 
+    [=AbortSignal/aborted flag=] is set to true, return [=a promise rejected with=] an "{{AbortError}}" 
```

Comment:
ditto

[webappsec-credential-management/#algorithm-request](https://w3c.github.io/webappsec-credential-management/#algorithm-request) is checking `options.signal`'s aborted flag.  same thoughts as above, i.e., `[[DiscoverFromExternalSource]]()` needs to promptly return and [webappsec-credential-management/#algorithm-request](https://w3c.github.io/webappsec-credential-management/#algorithm-request) ought to check immediately for abort situation?